### ISSUE TITLE
How to get everyone's ID in the prediction result

### ISSUE THREAD

**Issue by leafopening** [openpifpaf/openpifpaf#565]
Why can I get the parameter id_ only by using tshufflenetv2k30
How to get everyone's ID in the prediction result

**Comment by huyvt811** [comment]
Yes, I has same issue right here. Could you find something new @leafopening  ? I'm newbie and meet a trouble with openpifpaf. It's seems like not popular and community support
